controllers/authors.js

```diff
@@ -0,0 +1,32 @@
+const models = require('../models')
+
+const getAllAuthors = async (request, response) => {
+  const authors = await models.authors.findAll()
+
+  return response.send(authors)
+}
+
+const getAuthorWithNovelsAndGenresByidentifier = async (request, response) => {
+  const { identifier } = request.params
+
+  const authorNovelGenres = await models.authors.findOne({
+    include: [{
+      model: models.novels,
+      include: [{ model: models.genres }]
+    }],
+    where: {
+      [models.Op.or]: [
+        { id: { [models.Op.like]: `%${identifier}%` } },
+        { nameLast: { [models.Op.like]: `%${identifier}%` } },
+      ],
+    }
+  })
+
+
+
+  return authorNovelGenres
```

Comment:
Watch your spacing, it seems minor but can read to a potential employer as either "junior" or worse "careless"